pages/common/autoflake.md

```diff
@@ -0,0 +1,19 @@
+# autoflake
+
+> A tool to remove unused imports and variables from Python code.
+
+- Remove unused variables from a single file and display the diff:
+
+`autoflake --remove-unused-variables {{file.py}}`
+
+- Remove unused imports from multiple files and display the diffs:
+
+`autoflake --remove-all-unused-imports {{file1.py}} {{file2.py}} {{file3.py}}`
+
+- Remove unused variables from a file and make changes to that file:
+
+`autoflake --remove-unused-variables --in-place {{file.py}}`
+
+- Remove unused variables recursively from all files in a directory:
```

Comment:
Will this display the diff or change the file in-place ? I think we should add the `--in-place` flag to make it clear.